jwcrypto/jwe.py

```diff
@@ -492,7 +506,7 @@ def payload(self):
 
     @property
     def jose_header(self):
-        jh = self._get_jose_header()
+        jh = self._get_jose_header(self.objects.get('header'))
```

Comment:
This is a good catch, but would be nice to have it in a separate PR I can push right away,

Reply:
Ok sounds good, I will create a separate PR for this!